tutorials/getting_started/semantic_parsing.md

```diff
@@ -0,0 +1,315 @@
+# The AllenNLP Semantic Parsing Framework
+
+_Semantic parsing_ is the task of mapping language to some kind of formal meaning representation.
+This meaning representation could be a logical statement in lambda calculus or lambda-DCS, a set of
+instructions for a robot to follow, or even a python, java, or SQL program.  In some cases these
+meaning representations are directly executable in some environment (like executing a query against
+a database, or running python code), and in others they are simply an attempt to normalize the
+semantics of a natural language utterance (like Abstract Meaning Representations or open-domain CCG
+semantics).  The thing all of these variations have in common is that they try to capture the
```

Comment:
You may want to say that this tutorial focuses on the executable kind of semantic parsing.